web/src/pages/requests/index.tsx

```diff
@@ -232,10 +232,6 @@ const Home: NextPage = () => {
                     <Stack spacing={5} maxH="80vh">
                       {reqsUpcoming?.requests?.map((request, i) => (
                         <UserAccessCard
-                          onCancel={() => {
-                            mutatePast();
-                            mutate();
```

Comment:
I think these ones might be important, should be kept in? I understand that they revalidate the SWR hook for the detailed and list endpoints?

Reply:
This prop was removed from the user car but remained for some reason on this page.

The prop was originally removed because the "cancel" action was removed from the cards, so no need to mutate the list of requests by the user card.